docs/EN/Configuration/OmnipodEros.rst

```diff
@@ -441,37 +441,45 @@ Fields
 Icons
 -----
 
-.. list-table:: 
-      
-    * - |refresh_pod_status|
-      - **REFRESH:** 
-			
-	Sends a refresh command to the active pod to update communication
-			 
-	* Use to refresh the pod status and dismiss status fields that contain the text (uncertain).
-	* See the `Troubleshooting section <#troubleshooting>`__ below for additional information.
-    * - |pod_management|  	 
-      - **POD MGMT:**
-
-	Navigates to the Pod management menu   
-    * - |ack_alerts|		 
-      - **ACK ALERTS:**
-   			 
-	When pressed this will disable the pod expiration beeps and notifications. 
-			 
-	* Button is displayed only when pod time is past expiration warning time
-	* Upon successful dismissal, this icon will no longer appear.			 
-    * - |set_time|	 
-      - **SET TIME:**
-   
-	When pressed this will update the time on the pod with the current time on your phone.
-    * - |suspend|  		 
-      - **SUSPEND:**
-   
-	Suspends the active pod
-    * - |resume|	 
-      - **RESUME DELIVERY:**
-   
+* **REFRESH:**
+
+    |refresh_pod_status|
+
+    Sends a refresh command to the active pod to update communication
+
+    Use to refresh the pod status and dismiss status fields that contain the text (uncertain).
+
+    See the `Troubleshooting section <#troubleshooting>`__ below for additional information.
+
+* **POD MGMT:**
+
+    |pod_management|
+
+    Navigates to the Pod management menu
+
+* **ACK ALERTS:**
+
+    |ack_alerts|
+
+    When pressed this will disable the pod expiration beeps and notifications.
+
+    Button is displayed only when pod time is past expiration warning time
+    Upon successful dismissal, this icon will no longer appear.
+
+* **SET TIME:**
+
+    |set_time|
+
+    When pressed this will update the time on the pod with the current time on your phone.
+
+* **SUSPEND:**
+
+    |suspend|
+    Suspends the active pod
+
+* **RESUME DELIVERY:**
+
+    |resume|
 	Resumes the currently suspended, active pod
```

Comment:
Missing newline (otherwise inconsistent)

Reply:
This line currently renders as indented too.
![image](https://user-images.githubusercontent.com/9538230/161075966-94c3d002-fe46-4873-9b13-c2f4f0fa7526.png)
(see [here](https://androidaps.readthedocs.io/en/latest/Configuration/OmnipodEros.html#icons))